Lab/Challenge1.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lab
+{
+    public class Challenge1
+    {
+        public static int ReturnSum(int val1, int val2, int val3)
+        {
+            return val1 * val2 * val3;
+        }
+
+        public static void HandleChallenge1()
+        {
+            Console.WriteLine("Please enter 3 numbers");
+            string input = Console.ReadLine();
+            InputHandler(input);
+
+        }
+        public static int InputHandler(string input)
+        {
+            int sum;
+            string[] arr = input.Split(" ");
+            int[] intArr = new int[arr.Length];
+            try
+            {
+                for (int i = 0; i < arr.Length; i++)
+                {
+                    intArr[i] = Convert.ToInt32(arr[i]);
```

Comment:
Another way to handle this is to `try`/`catch` around just the `Convert.ToInt32()` rather than around the whole thing. This allows you to handle `"1 x 3"` giving 3 instead of 1.